docs/upgrade.html

```diff
@@ -19,6 +19,58 @@
 
 <script id="upgrade-template" type="text/x-handlebars-template">
 
+<h4><a id="upgrade_2_0_0" href="#upgrade_2_0_0">Upgrading from 0.8.x, 0.9.x, 0.10.0.x, 0.10.1.x, 0.10.2.x, 0.11.0.x, 1.0.x, 1.1.x, or 1.2.x to 2.0.0</a></h4>
```

Comment:
@apovzner @junrao while working on another PR, I realized this one duplicated the part of `upgrade_2_0_0` with `upgrade_1_2_0` (we renamed 1.2 to 2.0). If there is nothing new content added I'll go ahead and remove the duplicated section in my PR